### DISCUSSION TITLE
buttonTo with method "get" displays "?" in URL

### DISCUSSION THREAD

**Discussion by cfcoder** [wheels-dev/wheels#1376] [Q&A]
How do I get rid of the "?" when using a buttonTo method when using method="get"?  I realize buttons often are associated with forms which take a method of post or get with parameters. But in this case, I want a button to simply do a GET to open a new page. I don't need or want the "?" to show at the end of the URL.

![image](https://github.com/cfwheels/cfwheels/assets/4595721/ee86fd07-b954-43a4-94d1-4016f4850629)

**Comment by neokoenig** [comment]
Under the hood buttonTo actually creates (and submits) a form. In this instance I'd probably just use `linkTo()` and style the resulting `<a>` tag appropriately.
